README.md

```diff
@@ -13,7 +13,7 @@ The project is composed of 4 main parts:
 > - **Sam**, a set of command line tools.  
 
 [![Stories in Ready](https://badge.waffle.io/tuttleofx/tuttleofx.svg?label=1 - Ready)](http://waffle.io/tuttleofx/tuttleofx) <br />
-TuttleOFX is alive! 
+TuttleOFX is alive! <br/>
```

Comment:
br is not markdown.
To have a line break without a paragraph, you will need to use two trailing spaces.